developers/roadmap.md

```diff
@@ -18,8 +18,12 @@ version supported by the given release of the library:
     version that will not be released.*
 * `libsdformat8`:
     * 8.5.0 (**released**): C++17, SDFormat 1.6.
-* `libsdformat9`: For use in Ignition Gazebo.
+* `libsdformat9`: For use in Ignition Citadel.
```

Comment:
This information is duplicate below. Is the idea of writing "Fore use in..." here to explain what motivated the release? How about adding the same for the versions above? I believe it would be:

* 4 :arrow_right: Gazebo 7
* 6 :arrow_right: Gazebo 9
* 8 :arrow_right: Blueprint

Reply:
db3a35a3d45bdc1108aeda2f4c53d0a05bb54321